### DISCUSSION TITLE
Is there a config to remove the `OPEN BUFFERS` window?

### DISCUSSION THREAD

**Discussion by LintaoAmons** [nvim-neo-tree/neo-tree.nvim#1073] [Q&A] [answered]
There's three kinds of window, but I don't want the OPEN BUFFERS window

Is there a config to allow me remove that window?

![image](https://github.com/nvim-neo-tree/neo-tree.nvim/assets/95092244/fc8fab32-fb94-4e98-aea2-546aa0a69b7f)

**Comment by miversen33** [comment, marked as answer]
I mean, that completely depends on how you are using neo-tree. The simple answer is "don't use it". If you are using source selector,
you will want to read the help doc
`:h neo-tree-source-selector`

**Reply by LintaoAmons** [reply]
Thx~

```lua
    require("neo-tree").setup({
      source_selector = {
        sources = { -- table
          {
            source = "filesystem", -- string
            display_name = " 󰉓 Files ", -- string | nil
          },
          {
            source = "git_status", -- string
            display_name = " 󰊢 Git ", -- string | nil
          },
        },
      },
   ...
   }
```
